plugins/carla/cmake/rtmempool.cmake

```diff
@@ -0,0 +1,15 @@
+# base config
+set(carla_rtmempool_basedir carla/source/modules/rtmempool)
+set(carla_rtmempool_extra_libs ${carla_pthread_libs})
+
+# static lib
+add_library(carla-rtmempool STATIC)
```

Comment:
Does this have to be a static library? If the purpose is to "share" the source code among several other object, just use an `INTERFACE` library.